### ISSUE TITLE
Can you add rtp protocol 

### ISSUE THREAD

**Issue by hjchjcjh** [pedroSG94/RootEncoder#1573]
When I use RTSP and RTMP with high latency, my Android camera has poor streaming latency. It is best to use the RTP protocol to reduce latency, so can you add the RTP protocol? Thanks

**Comment by pedroSG94** [comment]
Hello,

Do you have a way to test it? 
I need an environment to test with RTP protocol

**Comment by hjchjcjh** [comment]
> Hello,
> 
> Do you have a way to test it? I need an environment to test with RTP protocol

Of course, you can use this app to display your RTP video stream for development testing
test app download：
https://drive.google.com/file/d/10LN0UyKvMGZju3FasQVBoWaQmL6UILx3/view?usp=sharing

![1725907309510](https://github.com/user-attachments/assets/0c48508a-2371-4fc6-a130-7a7ca798a4d8)
![f7c479ff3f678e8bac1d0f60210c6e8](https://github.com/user-attachments/assets/2e555c9a-e7d9-48fa-8298-a09efc27918c)

And  also can view the source code about this app:  https://github.com/Consti10/LiveVideo10ms

**Comment by pedroSG94** [comment]
I'm not able to connect that app to ffmpeg using rtp. Also, I can't find a server that allow RTP stream raw without RTSP. I think that implement this protocol if I don't know the usage of it is not a good idea.
This will be stopped until a clear way to test it is explained
